packages/jaeger-ui-components/src/TraceTimelineViewer/SpanBarRow.tsx

```diff
@@ -437,6 +441,30 @@ export class UnthemedSpanBarRow extends React.PureComponent<SpanBarRowProps> {
               </span>
               <small className={styles.endpointName}>{rpc ? rpc.operationName : operationName}</small>
             </a>
+            {createSpanLink &&
```

Comment:
We add the links here if they are specified. Bit ugly with the function wrapper and all so should be refactored later on. There are already some other links in the same space so this uses the same style.